integration_tests/tests/match/match.cy.ts

```diff
@@ -187,4 +188,46 @@ context('Placement Requests', () => {
     // Then I should see a success message on the list page
     Page.verifyOnPage(ListPage)
   })
+
+  it('allows me to book a space', () => {
+    // Given I am signed in as a cru_member
+    signIn(['cru_member'])
+
+    const premisesName = 'Hope House'
+    const premisesId = 'abc123'
+    const apType = 'normal'
+    const duration = 15
+    const startDate = '2024-07-23'
+
+    // And there is a placement request waiting for me to match
+    const person = personFactory.build()
+    const essentialCriteria: Array<PlacementCriteria> = ['isPIPE', 'acceptsHateCrimeOffenders']
+    const desirableCriteria: Array<PlacementCriteria> = ['isCatered', 'hasEnSuite']
+    const placementRequest = placementRequestDetailFactory.build({
+      person,
+      status: 'notMatched',
+      duration,
+      essentialCriteria,
+      desirableCriteria,
+    })
+
+    // When I visit the 'Book a space' page
+    cy.task('stubPlacementRequest', placementRequest)
+    const page = BookASpacePage.visit(placementRequest, startDate, duration, premisesName, premisesId, apType)
+
+    // Then I should see the details of the space I am booking
+    page.shouldShowBookingDetails(placementRequest, startDate, duration, apType)
+
+    // And when I complete the form
+    const spaceBooking = spaceBookingFactory.build({})
+    cy.task('stubSpaceBookingCreate', { placementRequestId: placementRequest.id, spaceBooking })
+    cy.task('stubPlacementRequestsDashboard', { placementRequests: [placementRequest], status: 'matched' })
+    page.clickSubmit()
+
```

Comment:
It would be good to assert the correct data has been sent to the API here similar to how to do for the search on line 145